crates/mdman/src/hbs.rs

```diff
@@ -139,6 +139,9 @@ impl HelperDef for OptionHelper<'_> {
         // Render the block.
         let block = t.renders(r, gctx, rc)?;
 
+        // Windows newlines can break some rendering, so normalize.
+        let block = block.replace("\r\n", "\n");
```

Comment:
Could you split this into two commits, test addition and fix, so the fix commit shows the behavior change through how the test changes?

Reply:
Done